File: demo_deblurring.m
% read and save images
img         = imread('1024_books_original.png');
%img         = imread('test_images/originals/1024_books_original.png');
img         = double(img)/255;

img(42,:)   = 0; 

figure; imshow(img);

imwrite(img,'meaning.png','png');

File: demo_deblurring.py
import numpy as np
from PIL import Image
from numpy import asarray
from matplotlib import image
from matplotlib import pyplot as plt


im = Image.open('test_images/1024_books_original.png') 
img = np.array(im)
img = img.astype(np.float64) / 255

m, n = img.shape
img[42,0:n-1] = 0

plt.figure(1)
plt.axis('off')
plt.gray()
plt.imshow(img)
plt.show()
plt.imsave('meaning.png', img)